src/annotator/anchoring/test/pdf-test.js

```diff
@@ -531,6 +531,11 @@ describe('annotator/anchoring/pdf', () => {
     });
 
     it('anchors with mismatch if text layer differs from PDF.js text API output', async () => {
+      const warnOnce = sinon.stub();
+      pdfAnchoring.$imports.$mock({
+        '../../shared/warn-once': { warnOnce },
+      });
```

Comment:
nb. There is no `$imports.$restore` call added in this PR to match this `$mock` call because one already exists in a higher-level `afterEach`.